_data/security.yml

```diff
@@ -237,6 +237,14 @@ websites:
       twitter: SaferVPN
       facebook: SaferVPNCom
 
+    - name: Sophos Central
+      url: https://central.sophos.com
```

Comment:
The main url seems to be `https://www.sophos.com`. Please change the value of the `url` tag to that.

Reply:
I can certainly change to the requested URL, however, they have several products and not all support MFA... Sophos Central is one that Does support MFA.